ts/Series/Organization/OrganizationPoint.ts

```diff
@@ -21,7 +21,7 @@
 import type OrganizationPointOptions from './OrganizationPointOptions';
 import type OrganizationSeries from './OrganizationSeries';
 import type { OrganizationSeriesNodeOptions } from './OrganizationSeriesOptions';
-import type SankeyPoint from './../Sankey/SankeyPoint';
+import SankeyPoint from './../Sankey/SankeyPoint';
```

Comment:
SankeyPoint is used as a type in this file, so please restore this change here. 

(If the full import is used, it increases the bundle size :) )